src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs

```diff
@@ -46,12 +46,19 @@ public static partial class PlatformDetection
         public static bool IsRedHatFamily7 => false;
         public static bool IsNotRedHatFamily6 => true;
 
+        // >= Windows 10 Anniversary Update
         public static bool IsWindows10Version1607OrGreater => 
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 14393;
+        
+         // >= Windows 10 Creators Update
         public static bool IsWindows10Version1703OrGreater => 
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 15063;
+        
+        // >= Windows 10 Fall Creators Update
         public static bool IsWindows10Version1709OrGreater => 
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 16299;
+        
+        // >= Windows 10 April 2018 Update
```

Comment:
Where did these comments come from?  Presumably they weren't auto-generated :)

Reply:
They were in the ref. I removed it from there so added them to the source.